### ISSUE TITLE
How to use Manager->PressButton equivalent ?

### ISSUE THREAD

**Issue by gelleouet** [OpenZWave/node-openzwave-shared#41]
Hi, 
I use this library and it works perfectly with a raspberry. 
I try to control a fibaro fgrm 222 : switchOn, switchOff and setValue work fine. 
This module has values with genre='user' and type='button' (label = "Close", "Open", etc.)
I would like to know how control this value : i try zwave.beginControllerCommand("Open", nodeId) but it throw an exception "undefined is not a function".

Is it possible to control this values with this library ?
tks

**Comment by ekarak** [comment]
beginControllerCommand() is not what controls these values, but only setValue().
Simply put, support for controlling buttons is missing :)
I'll add it in the weekend.

**Comment by gelleouet** [comment]
Thanks for your reply.
With this feature, i could make simpler my smarthome system !
Thks

**Comment by ekarak** [comment]
can you please download the 'buttons' branch and try compiling it? 
https://github.com/OpenZWave/node-openzwave-shared/archive/buttons.zip

**Comment by gelleouet** [comment]
Ok but there is a comment in openzwave-nodes.cc. 
```
// TODO:
// OpenZWave::Manager::Get()->PressButton(valueID);
```

**Comment by ekarak** [comment]
You're right :) sorry about this. I've just pushed the proper code, can you try again?

**Comment by ekarak** [comment]
usage is:
`zwave.pressButton(OpenZWave::valueId);` 
or you can pass individual value constituents:
`zwave.pressButton(nodeid, commandclass, instance, index);`

**Comment by gelleouet** [comment]
I compile the buttons branch et try new "pressButton" with success. It has worked after a complete raspberry reboot (?) and only the second method with individual values work for me. Maybe my value object wasn't correct.

Thanks a lot for your commit !

**Comment by ekarak** [comment]
Thanks Gregory for the feedback!
I'm keeping issue open until the merge to master is complete. I want to test this myself (when I have the time)

**Comment by ekarak** [comment]
closing - released version 1.1.5
